js/ai/src/extract.ts

```diff
@@ -15,6 +15,7 @@
  */
 
 import JSON5 from 'json5';
+import untruncateJson from 'untruncate-json';
```

Comment:
We have a sample here using the `partial-json` package:
https://github.com/firebase/genkit/blob/1f6534b2a586006317699345935ca8268ddc6a1d/samples/js-angular/server/src/jsonStreaming.ts#L20C31-L20C43

Any obvious differences between those two?